cli/cmd/inject.go

```diff
@@ -56,10 +57,21 @@ with 'conduit inject'. e.g. curl http://url.to/yml | conduit inject -
 				return err
 			}
 		}
-		return InjectYAML(in, os.Stdout)
+		return runInjectCmd(in, os.Stderr, os.Stdout)
```

Comment:
I see that if you want to test the output to stderr then my suggestion doesn't work. In that case you can just have runInjectCmd return the exit code for `os.Exit()`, so that you can write:

```go
exitCode := runInjectCmd(in, os.Stderr, os.Stdout)
os.Exit(exitCode)
return nil
```